src/app/(auth)/register/page.tsx

```
@@ -77,17 +95,48 @@ export default function Register() {
           })}
           required
         />
+        <TextField
+          className="col-span-full"
+          label="电话"
+          name="telephone"
+          type="telephone"
+          autoComplete="tel"
+          value={registerInfo.telephone}
+          onChange={(e) => setRegisterInfo({
+            ...registerInfo,
+            telephone: e.target.value
+          })}
+          required
+        />
+        <TextField
+          className="col-span-full"
+          label="邮箱"
+          name="email"
+          type="email"
+          autoComplete="email"
+          value={registerInfo.email}
+          onChange={(e) => setRegisterInfo({
+            ...registerInfo,
+            email: e.target.value
+          })}
+          required
+        />
         <div className="col-span-full">
-          <Button 
-            type="submit" 
-            variant="solid" 
-            color="blue" 
-            className="w-full"
-          >
-            <span>
-              注册 <span aria-hidden="true">&rarr;</span>
-            </span>
-          </Button>
+          <Link
```

Due to the problem mentioned above, the button should not be a simple link. We should throw an exception if the username is already taken.